qa/L0_batcher/batcher_test.py

```diff
@@ -1895,7 +1897,7 @@ def test_preferred_batch_only_use_no_preferred_size(self):
     def test_max_queue_delay_only_non_default(self):
         # Send 12 requests with batch size 1. The max_queue_delay is set
         # to non-zero. Depending upon the timing of the requests arrival
-        # there can be either 1 or 2 model executions.
+        # there can be either 1 or multiple model executions.
```

Comment:
Can you explain why the originally intended behavior is not consistent and needs to be relaxed here? (different for zero and non-zero queue delay scenarios?)